packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -77,182 +88,205 @@ describe('ui:create:atomic', () => {
+        ).toBeTruthy();
+      }, 60e3);
+    });
 
-      expect(
-        JSON.parse(await (await tokenResponseListener).text())
-      ).toMatchObject({
-        token: expect.stringMatching(jwtTokenPattern),
-      });
-    }, 60e3);
+    describe('when the default Stencil port is busy', () => {
+      let dummyServer: DummyServer;
+      let serverProcessManager: ProcessManager;
 
-    it('should send a search query when the page is loaded', async () => {
-      await page.goto(searchPageEndpoint, {waitUntil: 'networkidle2'});
-      await page.waitForSelector(searchInterfaceSelector);
+      beforeAll(async () => {
+        serverProcessManager = new ProcessManager();
+        processManagers.push(serverProcessManager);
 
-      expect(interceptedRequests.some(isSearchRequestOrResponse)).toBeTruthy();
-    }, 60e3);
-  });
+        dummyServer = new DummyServer(3333);
 
-  describe('when the default Stencil port is busy', () => {
-    let dummyServer: DummyServer;
-    let serverProcessManager: ProcessManager;
+        const appTerminal = await startApplication(
+          serverProcessManager,
+          'stencil-port-test'
+        );
+        await waitForAppRunning(appTerminal);
+      }, 2 * 60e3);
 
-    beforeAll(async () => {
-      serverProcessManager = new ProcessManager();
-      processManagers.push(serverProcessManager);
+      afterAll(async () => {
+        await dummyServer.close();
+        await serverProcessManager.killAllProcesses();
+      }, 30e3);
 
-      dummyServer = new DummyServer(3333);
+      it('Netlify should still load the Stencil app properly', async () => {
+        await page.goto(searchPageEndpoint, {
+          waitUntil: 'networkidle2',
+        });
 
-      const appTerminal = await startApplication(
-        serverProcessManager,
-        'stencil-port-test'
-      );
-      await waitForAppRunning(appTerminal);
-    }, 2 * 60e3);
+        expect(await page.$(searchInterfaceSelector)).not.toBeNull();
+      }, 60e3);
+    });
+  }
 
-    afterAll(async () => {
-      await dummyServer.close();
-      await serverProcessManager.killAllProcesses();
-    }, 30e3);
+  describe('when using the default page config (pageId not specified)', () => {
+    setupTests({});
+    runTests();
+  });
 
-    it('Netlify should still load the Stencil app properly', async () => {
-      await page.goto(searchPageEndpoint, {
-        waitUntil: 'networkidle2',
-      });
+  describe('when using an existing pageId (--pageId flag specified)', () => {
+    // TODO: CDX-969 update search page id from new stg org.
+    setupTests({pageId: '85fe78b4-2e10-4ed9-a0b7-664f5d23887d'});
+    runTests();
+  });
 
-      expect(await page.$(searchInterfaceSelector)).not.toBeNull();
-    }, 60e3);
+  describe.skip('when using an existing pageId (using the list prompt of available pages)', () => {
+    // TODO: CDX-969 make sure stg org has at least 1 "new" hosted search page.
+    setupTests({promptAnswer: `\033[B${EOL}`});
```

Comment:
I really have no clue how to enter an "arrow down" to select one of the search pages 🤔

Reply:
A Node terminal works like a VT100, here's a manual: https://vt100.net/docs/vt100-ug/chapter3.html
There's an escape sequence that matches the down arrow key or the down arrow key action (i.e. move cursor 1 line down).
You need to write that in the stdin and voilà

Reply:
So 
```suggestion
    setupTests({promptAnswer: `ESC D${EOL}`});
```
?

Reply:
My original answer gave me this 
<img width="1103" alt="Screen Shot 2022-05-30 at 9 23 35 AM" src="https://user-images.githubusercontent.com/4923043/171009970-b23eb717-7870-44da-8a59-7a902a2744ee.png">


`\x1B[B` doesn't go down 1 line